jenkins/files/Debian/jenkins.conf

```diff
@@ -8,18 +8,20 @@ NAME=jenkins
 JAVA=/usr/bin/java
 
 # arguments to pass to java
-JAVA_ARGS="-Djava.awt.headless=true"  # Allow graphs etc. to work even when an X server is present
+#JAVA_ARGS="-Djava.awt.headless=true"  # Allow graphs etc. to work even when an X server is present
```

Comment:
I know there is git history, but it is the part of the original config file, so I left it in as a reference should someone want to edit the file by hand. Like every other commented-out option.